indra/llui/llfloater.cpp

```diff
@@ -602,31 +586,27 @@ LLControlGroup*	LLFloater::getControlGroup()
 void LLFloater::setVisible( BOOL visible )
 {
 	LLPanel::setVisible(visible); // calls onVisibilityChange()
-	if( visible && mFirstLook )
+	if (visible && mFirstLook)
 	{
 		mFirstLook = FALSE;
 	}
 
-	if( !visible )
+	if (!visible)
 	{
 		LLUI::getInstance()->removePopup(this);
 
-		if( gFocusMgr.childHasMouseCapture( this ) )
+		if (gFocusMgr.childHasMouseCapture(this))
 		{
 			gFocusMgr.setMouseCapture(NULL);
 		}
 	}
 
-	for(handle_set_iter_t dependent_it = mDependents.begin();
-		dependent_it != mDependents.end(); )
+	for (const LLHandle<LLFloater>& handle : mDependents)
 	{
-		LLFloater* floaterp = dependent_it->get();
```

Comment:
Personaly I would have preferred old variant, it is cleaner (I suggest checking if this matches out coding policy).